presto-main/src/main/java/com/facebook/presto/operator/scalar/RowIndeterminateOperator.java

```diff
@@ -132,15 +132,14 @@ public ScalarFunctionImplementation specialize(BoundVariables boundVariables, in
                                 .push(true)
                                 .gotoLabel(end));
 
-                Signature signature = internalOperator(
-                        INDETERMINATE.name(),
-                        BOOLEAN.getTypeSignature(),
-                        ImmutableList.of(fieldTypes.get(i).getTypeSignature()));
-                ScalarFunctionImplementation function = functionManager.getScalarFunctionImplementation(signature);
+                FunctionHandle functionHandle = functionManager.resolveOperator(
+                        INDETERMINATE,
+                        TypeSignatureProvider.fromTypeSignatures(fieldTypes.get(i).getTypeSignature()));
+                ScalarFunctionImplementation function = functionManager.getScalarFunctionImplementation(functionHandle);
                 BytecodeExpression element = constantType(binder, fieldTypes.get(i)).getValue(value, constantInt(i));
 
                 ifNullField.ifFalse(new IfStatement("if the field is not null but indeterminate...")
-                        .condition(invokeFunction(scope, cachedInstanceBinder, signature.getName(), function, element))
+                        .condition(invokeFunction(scope, cachedInstanceBinder, functionHandle.getSignature().getName(), function, element))
```

Comment:
You can probably use `mangleOperatorName(INDETERMINATE)` to get the name here. There's no need to resolve the operator here.